pkg/router/router.go

```diff
@@ -4,7 +4,7 @@ import (
 	"net/http"
 
 	"github.com/AsterNighT/software-engineering-backend/api"
-	_ "github.com/AsterNighT/software-engineering-backend/docs" // swagger doc
+	//	_ "github.com/AsterNighT/software-engineering-backend/docs" // swagger doc
```

Comment:
Pls do not comment out this, keep docs in your folder and ignore it rather than delete it